### ISSUE TITLE
Questions about ignite damage. 

### ISSUE THREAD

**Issue by Wearetestingstuff** [LightsHope/server#1307]
First of all, thank you for taking the time to report this issue.

Please note that the bug tracker does not accept reports on account or character
issues that are unrelated to the server's development. You should open a ticket
in game and wait for the response of a gamemaster. They may redirect you to this
bug tracker if they believe there's a server issue that they cannot help you
with.

Before writing your report, please do read and consider the following points to
increase the quality of your report.

- Search the issue tracker for open and closed issues before opening a new
  issue. You may be able to contribute additional facts by commenting on
  existing issues instead of adding extra noise in a separate issue.

- http://www.wowhead.com/ and http://db.vanillagaming.org/ are not valid sources
  for quests, items etc., instead, please use archived versions of
  http://wow.allakhazam.com or http://www.thottbot.com through
  https://archive.org/.

- Validate the date and time of the archived sources to a certain patch. The
  patch release dates can be found on the corresponding WoWWiki article, as
  found at http://wowwiki.wikia.com/wiki/Category:World_of_Warcraft_patches.

- Videos are valid if and only if the footage contains a timestamp or
  information that confirms the time it was recorded. Date of upload is NOT
  valid and does not confirm when it was shot.

- Read the Effective Bug Reporting article at
  https://github.com/elysium-project/server/blob/development/docs/CONTRIBUTING.md
  which covers these points and more.

Once you've read this message, delete it and write your report below.

--

I've been testing the ignite damage and something seems to be off. I know that the method that was originally introduced on anathema and other private servers was not blizzlike but the bug that has been mentioned on the wowiki and such does not seem to be repeatable. There were instances of ignites adding together and causing more damage than the capped amount at 5 stacks.

Here is an archived post talking about "rolling" ignites from 2006.

![ignites](https://user-images.githubusercontent.com/35764566/35328081-d87855a4-00c0-11e8-8657-6bc04a14671a.PNG)

If consider that the mage was casting fireballs the entire 22 seconds and did approx 10k damage in fireballs that means the ignite was still ticking over 7000 damage or 14,000 dmg per 4 seconds. With Full buffs power infusion and such the biggest ignite tick i could get around 3400.

**Comment by Dalleren** [comment]
http://wowwiki.wikia.com/wiki/Ignite_(old)

Theoretically in Vanilla it can stack infinitely if reset before it expires (after 2 seconds or two tics) and you have an infinitely long boss fight. This means that a mage must crit with a fire spell before 4 seconds have passed, otherwise Ignite will expire and a new Ignite is applied whenever a Mage crits with a fire spell.

This means that Ignite will only do damage when it tics (at 2 seconds and 4 seconds), and can after a long encounter and managed correctly result in very high DoT tics.

Practically you would achieve this by either having 2-3 Mages using Scorch (1.5 second cast time) or enough mages so that everyone can use Fireball, and praying to god that you can crit at least once every 4 seconds.

**Comment by GuybrushGit** [comment]
We looked into this in depth during Elysium but all the research is now gone.
Ignite damage does not increase infinitely, after 5 stacks all crits do is refresh the duration. 
The high numbers are from fights with damage modifiers like Patchwerk.

**Comment by Wearetestingstuff** [comment]
I don't think that it was ever able to stack infinity but from what i've been reading a lot of people talk about stealing the ignites which might have caused a bug to cause it to then overlap and start stacking or "rolling". I obviously have no proof of this but I am certain that the ignite we have now on lights hope is not "blizzlike".
